src/styles/article-reader.scss

```diff
@@ -0,0 +1,85 @@
+.article-reader {
+  max-width: 768px;
+  padding-left: 4.2rem;
+  padding-top: var(--space-80);
+  width: calc(100% - 36rem);
+
+  hr {
+    border: none;
+    border-bottom: 0.4rem solid var(--black2);
+  }
+}
+
+/* sr-only for desktop */
+
+.article-reader__headline {
+  padding: 0;
+}
+
+.hero h1 {
+  cursor: pointer;
+}
+
+.article-reader {
+  blockquote {
+    border-left: 3px solid var(--black4);
+    color: var(--color-text-primary);
+    font-style: italic;
+    font-weight: 200;
+    letter-spacing: 1px;
+    margin: 0 0 2.8rem 0;
+    padding: 1.8rem 0 1.8rem 3.2rem;
+
+    p {
+      margin: 0;
+      padding: 0;
+    }
+  }
+
+  section {
+    margin-bottom: var(--space-48);
+  }
+
+  ul {
+    margin-bottom: var(--space-24);
+  }
+
+  li {
+    + h4 {
+      margin-top: var(--space-32);
+    }
+
+    margin-bottom: var(--space-08);
+  }
+
+  h3 {
+    margin-bottom: var(--space-24);
+  }
+
+  h4 {
+    margin-bottom: var(--space-16);
+  }
+
+  hr {
+    margin-bottom: var(--space-40);
+  }
+
+  details,
+  iframe {
+    margin-bottom: var(--space-24);
+  }
+
+  p + {
+    h2 {
+      margin-top: var(--space-40);
+    }
+
+    h3 {
+      margin-top: var(--space-32);
+    }
+  }
+
+  .api-key__section {
+    /* margin-bottom: 0; */
```

Comment:
We probably want this class removed if its commented out

Reply:
Seems like that, but I don't want to mix changes in one PR. We can create another PR with styles refactoring, after this will be merged

Reply:
I converted CSS to SCSS "as it is"

Reply:
I agree